test/lib/runtime/questionnaire_action_test.exs

```diff
@@ -0,0 +1,64 @@
+defmodule Ask.QuestionnaireExportTest do
+  use Ask.ModelCase
+  alias Ask.Runtime.QuestionnaireExport
+
+  describe "clean_i18n_entity" do
+    test "base case" do
+      entity = %{"en" => "foo", "es" => "bar"}
+
+      clean = QuestionnaireExport.clean_i18n_entity(entity, ["en"], "")
+
+      assert clean == %{"en" => "foo"}
+    end
+
+    test "move forward" do
+      entity = %{"en" => "foo", "es" => "bar"}
+
+      clean = QuestionnaireExport.clean_i18n_entity(entity, ["en"], ".")
```

Comment:
There is not such thing as move forward. `.property` or `.[]` should be consumed. empty path is a base.
But there is no `.` alone. In the current implementation `.....property` = `.property` = `property` which make no sense because only `.property` is a JSON path expression as in jq.